server/models/interfaces.ts

```diff
@@ -22,6 +22,7 @@ import {
   Rollup,
   Transform,
 } from "../../models/interfaces";
+import { getIndexToDataStreamMapping } from "../services/DataStreamService";
```

Comment:
Not used?

Reply:
Not related to snapshot restore, but I will get rid of it.